requirements.txt

```diff
@@ -1,27 +1,27 @@
 Django==1.11.3
 Whoosh==2.7.4
-django-auth-ldap==1.2.8
+django-auth-ldap==1.2.13
 django-form-utils==1.0.3
-django-haystack==2.6.0
-django-reversion==2.0.8
-django-tastypie==0.13.3
+django-haystack==2.6.1
+django-reversion==2.0.9
+django-tastypie==0.14.0
```

Comment:
Is this upgrade extensively tested? Wouldn't be the first time tastypie would break.